talks/from-chat-to-agent/styles/index.css

```
@@ -0,0 +1,592 @@
+  left: 110px;
+  right: 110px;
+  transform: translateY(-50%);
+  display: flex;
+  justify-content: center;
+}
+.evolution__bar {
+  display: none;
+}
+.evolution__timeline {
+  --timeline-inset: 58px;
+  position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)